src/charm.py

```diff
@@ -186,6 +187,14 @@ def _on_certificate_creation_request(self, event: CertificateCreationRequestEven
         )
         logger.info(f"Generated certificate for relation {event.relation_id}")
 
+    def _on_get_ca_cert(self, event):
```

Comment:
Please add typing for the `event` (probably ActionEvent)

Reply:
Done.